versions.props

```diff
@@ -66,7 +66,7 @@ org.eclipse.jetty.toolchain:jetty-servlet-api=4.0.6
 org.glassfish.hk2*:*=3.1.1
 org.glassfish.hk2.external:jakarta.inject=2.6.1
 org.glassfish.jersey*:*=3.1.11
-org.glassfish.jersey.containers:jersey-container-jetty-http=2.39.1
+org.glassfish.jersey.containers:jersey-container-jetty-http=2.47
```

Comment:
`main` looks to be still on 2.39.1 currently: https://github.com/apache/solr/blob/80fa70f2ccf2a4fd03658d822a5ffab35819119f/gradle/libs.versions.toml#L138

Reply:
#3108 for `main` - presuming we mark this PR as draft then in the meantime.

Reply:
I believe that we want to update `main` to 2.47 first before looking into upgrading `main` to 3 and leaving this with the backport of `main` at 2.47?